tools/src/test/java/org/apache/kafka/tools/MetadataQuorumCommandTest.java

```diff
@@ -172,4 +173,22 @@ public void testDescribeQuorumInZkMode() {
         );
 
     }
+
+    @ClusterTests({
+        @ClusterTest(clusterType = Type.CO_KRAFT, brokers = 1, controllers = 1),
+    })
+    public void testHumanReadableTimestamps() {
```

Comment:
this test doesn't test the business logic that we introduced in the code, i.e. the part about ensuring X in "X ms ago" is what is desired is missing.